opentracing/standard/raw.py

```diff
@@ -30,13 +30,13 @@ def __init__(self, trace_context, operation_name):
         self.operation_name = operation_name
         self.start_time = None
         self.duration = None
-        self.tags = None  # array of (key,vale) pairs when initialized
+        self.tags = None  # dict when initialized
         self.logs = None  # array of RawLog's when initialized
 
-    def add_tag(self, key, value):
+    def set_tag(self, key, value):
         if self.tags is None:
-            self.tags = []
-        self.tags.append((key, value))
```

Comment:
in a few implementations of tracers I've seen, they all _append_ tags to the list, rather than keep a map.

But the 'standard' tracer is just a prototype imo, a starting point for people to build their own tracer, so I don't have strong feelings about what it does. I am kind of in favor of lowering its status to a "template" or "skeleton", or whatever we want to call it, rather than implying that this is a production-quality code that people can extend and build real tracers on top of. We should only provide an API, not support implementations.  And after the API simplification, it's not clear if there's even value in having an extensible standard tracer framework.